src/Compilers/Core/Portable/CodeGen/PrivateImplementationDetails.cs

```diff
@@ -23,17 +23,16 @@ internal sealed class PrivateImplementationDetails : DefaultTypeDef, Cci.INamesp
     {
         // Note: Dev11 uses the source method token as the prefix, rather than a fixed token
         // value, and data field offsets are unique within the method, not across all methods.
-        private const string MemberNamePrefix = "$$method0x6000001-";
-        internal const string SynthesizedStringHashFunctionName = MemberNamePrefix + "ComputeStringHash";
+        internal const string SynthesizedStringHashFunctionName = "ComputeStringHash";
 
-        private readonly Cci.IModule _module;                     //parent unit
+        private readonly Cci.IModule _moduleBuilder;                     //parent unit
```

Comment:
I'd rather remove it, as it is not true. This is the module builder, not the module. This was terribly confusing to me when I was working on this code. I'd love to disentangle the two types.